dotcom-rendering/src/components/Carousel.importable.tsx

```diff
@@ -164,34 +190,38 @@ const carouselStyle = css`
 	}
 `;
 
-const dotsStyle = css`
+const dotsSmallMargin = css`
```

Comment:
For single, dynamic declarations like this, it might be more appropriate to use the `style` prop, [as per Emotion best practices](https://emotion.sh/docs/best-practices#use-the-style-prop-for-dynamic-styles).